test/stdlib/UnicodeScalarPropertiesTests.swift

```diff
@@ -83,7 +83,7 @@ UnicodeScalarPropertiesTests.test("properties.booleanProperties") {
   expectBooleanProperty(\.changesWhenNFKCCaseFolded, trueFor: "A", falseFor: "!")
 
 #if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
-  if #available(iOS 10.2, *) {
+  if #available(macOS 10.12, iOS 10, tvOS 10, watchOS 3, *) {
```

Comment:
iOS10.2 added new emoji beyond iOS10, which can affect the results of this test. I think the .2 needs to stay.

Otherwise, LGTM

Reply:
@milseman Should I use the other equivalent platform versions?

```
#available(macOS 10.12.2, iOS 10.2, tvOS 10.1, watchOS 3.1.1, *)
```

Should I also change the `@available` platform versions?

Reply:
@milseman I've changed the availability to the [December 2016](https://blog.emojipedia.org/ios-10-2-emoji-changelog/) platform updates.

Reply:
I think the point was that the *properties* were available but the *test* behavior changed.